docs_user/modules/proc_adopting-the-identity-service.adoc

```diff
@@ -72,6 +72,12 @@ done
 
 .Verification
 
+. Wait for the `OpenStackControlPlane` resource to become `Ready`:
```

Comment:
This should be after `openstack endpoint list | grep keystone` as in test, or test should be fixed to be consistent

Reply:
Yea fair enough it probably makes sense to check the keystone-specific functionality first and only then see if the control plane as a whole converged.